src/Splat/Platforms/Android/Bitmaps/PlatformBitmapLoader.cs

```diff
@@ -127,5 +129,29 @@ public IBitmap Create(float width, float height)
 
             return result;
         }
+
+        internal static Type[] GetTypesFromAssembly(Assembly assembly)
+        {
+            try
+            {
+                // could this be assembly.GetExportedTypes() ?
+                return assembly.GetTypes();
```

Comment:
Are there any cases where there are internal resources auto-generated, eg like the resource generation in wpf or winforms? That's the only scenario I can think of where we might not want to use GetExportedTypes().

Reply:
will have to produce a test as I don't know if the android resources are public. simple test to enumerate them and if they are public etc.
need to check the platform loader for wpf etc anyway on .net core i will raise an issue for that

Reply:
With .NET Core WPF there is a branch we been using for that since it's officially not out yet.

Reply:
https://github.com/reactiveui/splat/blob/preview/8.0-dotnet/src/Splat/Platforms/netcoreapp3/Bitmaps/PlatformBitmapLoader.cs

Reply:
cool it's probably all in hand then :)